main/forkScript.js

```diff
@@ -22,15 +23,31 @@ require('../lib/confluence/metric_computer_service.es6.js');
 require('../lib/dao/dao_container.es6.js');
 require('../lib/dao/http_json_dao.es6.js');
 require('../lib/dao/local_json_dao.es6.js');
+require('../lib/web_apis/api_compat_data.es6.js');
 require('../lib/web_apis/release.es6.js');
 require('../lib/web_apis/release_interface_relationship.es6.js');
 require('../lib/web_apis/web_interface.es6.js');
 require('../lib/web_catalog/api_extractor_service.es6.js');
 const pkg = org.chromium.apis.web;
 
 const logger = foam.log.ConsoleLogger.create();
-const container = pkg.DAOContainer.create(null, logger);
-foam.box.node.ForkBox.CONNECT_TO_PARENT(foam.box.Context.create({
-  unsafe: false,
-  classWhitelist: require('../data/class_whitelist.json'),
-}, container));
+
+const compatClassFile = 'class:org.chromium.apis.web.generated.CompatData.json';
+
+// TODO(markdittmer): This should be local or remote based on param passed to
+// parent. It should be forwarded to forkScript invocation.
+const compatClassURL =
+    `${require('../data/http_json_dao_base_url.json')}/${compatClassFile}`;
```

Comment:
If it's just a string const, probably export its value as a sensible name, as the behaviour of the require/import.

Reply:
Yeah. I think there are release process reasons this was isolated into a JSON file. I already have a TODO to change the way this line of code works anyway, so I'm going to leave it for now.